cmd/nodecmd/create.go

```diff
@@ -1043,18 +1016,29 @@ func generateNodeCertAndKeys(stakerCertFilePath, stakerKeyFilePath, blsKeyFilePa
 }
 
 func provideStakingCertAndKey(host *models.Host) error {
-	instanceID := host.GetCloudID()
-	keyPath := filepath.Join(app.GetNodesDir(), instanceID)
+	keyPath := filepath.Join(app.GetNodesDir(), "staking", host.IP)
```

Comment:
I suggest we create a separate func that uses `keyPath := filepath.Join(app.GetNodesDir(), "staking", host.IP)` so IP is used by default when we want to save stuff. In terms of the reading stuff I suggest func that used `keyPath := filepath.Join(app.GetNodesDir(), "staking", host.IP)` but falls back to keyPath := filepath.Join(app.GetNodesDir(), "staking", NodeID) if this folder doesn't exists to support backward compatibility with nodes created with prev versions of CLI